src/Momento.Sdk/Internal/DataGrpcManager.cs

```diff
@@ -256,10 +250,13 @@ public class DataGrpcManager : GrpcManager
 
     internal DataGrpcManager(IConfiguration config, string authToken, string endpoint): base(config.TransportStrategy.GrpcConfig, config.LoggerFactory, authToken, endpoint, "DataGrpcManager")
     {
+        var readConcernHeader = new Header(Header.ReadConcern, config.ReadConcern.ToStringValue());
+        headers.Add(readConcernHeader);
```

Comment:
Do we need to add the read concern header if it's set to balanced? It seems like extra bytes over the wire for no difference. I suppose the default may change in the future.

Reply:
i agree with your comment @nand4011 , I would omit it unless it's explicitly specified in the config